doc/man3/ASYNC_WAIT_CTX_new.pod

```diff
@@ -103,14 +120,47 @@ code that the job should be resumed by making the wait file descriptor
 "readable". Once resumed the engine should clear the wake signal on the wait
 file descriptor.
 
+Besides file descriptor, an optional additional method of notification is
```

Comment:
Replace first sentence with: As well as file descriptors, user code may also be notified via a callback.

Reply:
done